charts/nr-ebpf-agent/README.md

```diff
@@ -6,29 +6,16 @@ A Helm chart to monitor a Kubernetes Cluster using the eBPF agent.
 
```

Comment:
Please create a `README.md.gotmpl` that templates this README. As an example, please see this file: https://github.com/newrelic/helm-charts/blob/master/charts/nr-k8s-otel-collector/README.md.gotmpl